scripts/installFFDriver.js

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env node
+const { runScript } = require('./util');
+
+const setupDrivers = async function () {
+  try {
+    await runScript('./node_modules/protractor/bin/webdriver-manager', 'update --chrome=false'.split(' '));
+    console.log('FirefoxDriver Installed');
+  } catch (err) {
+    console.log(' ');
+    console.log('!!!!!!!!!!!-----------IMPORTANT----------!!!!!!!!!!!!!!!');
+    console.log('It looks like it hasn’t install properly, you may be behind a corporate proxy. You may have to add the --proxy flag to webdriver-manager in your package json.');
```

Comment:
hasn't installed*

Reply:
?

Reply:
It's a grammar error.
```
It looks like it hasn’t install properly
```
should be
```
It looks like it hasn’t installed properly
```

Reply:
Oh wow, I looked at that like 3 times (under the weather)